specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2020-12-08/healthbot.json

```diff
@@ -181,7 +181,8 @@
               "$ref": "#/definitions/Error"
             }
           }
-        }
+        },
```

Comment:
Does it need to remove the part of "200" in "responses"?

Reply:
After discussion, we decided to stay with the 200 response code along with the 201.